app/src/androidTest/java/cz/muni/fi/pv256/movio2/uco_396537/DatabaseManagerTest.java

```diff
@@ -0,0 +1,91 @@
+package cz.muni.fi.pv256.movio2.uco_396537;
+
+import android.test.AndroidTestCase;
+
+import java.util.List;
+
+import cz.muni.fi.pv256.movio2.uco_396537.Database.DatabaseContract;
+import cz.muni.fi.pv256.movio2.uco_396537.Database.DatabaseManager;
+import cz.muni.fi.pv256.movio2.uco_396537.Models.Movie;
+
+/**
+ * Created by david on 8.1.17.
+ */
+
+public class DatabaseManagerTest extends AndroidTestCase {
+
+    private DatabaseManager mManager;
+
+    @Override
+    protected void setUp() throws Exception {
+        mManager = new DatabaseManager(mContext);
+    }
+
+    @Override
+    public void tearDown() throws Exception {
+        mContext.getContentResolver().delete(
```

Comment:
Vzhledem k tomu, že i Jonáš Ševčík, to tak měl v ukázkovém projektu. A ani v tutoriálech jsem nenašel jiný způsob, jak by bylo možné mocknout nebo testovat nad jinou databází, nechal jsem to takhle. Data z databáze si však před testem zálohuji  a po testu vrátím zpět, aby byly zachovány.